components/automate-cli/pkg/verifyserver/models/startMockServer.go

```diff
@@ -0,0 +1,32 @@
+package models
+
+import (
+	"net"
+	"net/http"
+)
+
+// StartMockServerRequestBody contains the configuration for starting a mock server.
+type StartMockServerRequestBody struct {
+	Port     int
+	Protocol string
+	Cert     string
+	Key      string
+}
+
+// NewStartMockServerRequestBody creates a new StartMockServerRequestBody object.
+func NewStartMockServerRequestBody(port int, protocol string, cert string, key string) *StartMockServerRequestBody {
```

Comment:
We're not using this method, so please remove this. If you want to still keep it, please move it to any of the handlers. This is not the correct place for this method.

Reply:
Removed